float64 speed
float64 omega

float64 height
float64 height_vel

float64 roll
float64 pitch
float64 pitch_vel